### ISSUE TITLE
cargo outdated does not work with dependency packages

### ISSUE THREAD

**Issue by kjeremy** [kbknapp/cargo-outdated#146]
`cargo outdated` fails on
```toml
[dependencies]
hir = { path = "../ra_hir", package = "ra_hir" }
```

with
> error: no matching package named `hir` found (required by `ra_analysis`)
location searched: file:///C:/projects/test/rust-analyzer/crates/ra_hir
version required: *

See https://github.com/rust-analyzer/rust-analyzer/issues/249#issuecomment-442461852

**Comment by repi** [comment]
Ran into this issue as well in our large workspace at Embark where we do have a few packages that we rename with `package = "bla"` to simplify usage within crates.

**Comment by kornelski** [comment]
fixed in #161
